tests/unit/test_simulation_service.py

```
@@ -306,12 +306,17 @@ async def test_execute_test_script_unknown_action(self):
         """Test environment validation success."""
-        with patch('services.simulation_service.playwright'):
+        with patch('builtins.__import__'):
             result = self.service.validate_environment()
             assert result is True
     
     def test_validate_environment_failure(self):
         """Test environment validation failure."""
-        with patch('services.simulation_service.playwright', side_effect=ImportError):
+        def mock_import(name, *args):
+            if name == 'playwright':
+                raise ImportError("Playwright not found")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)